packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -7690,8 +7698,21 @@ export default class Meeting extends StatelessWebexPlugin {
       shareVideoEnabled = true,
       remoteMediaManagerConfig,
       bundlePolicy = 'max-bundle',
+      additionalMediaOptions,
```

Comment:
@rajithaBK, I believe this should fix it - but would need you to test it on the sdk sample app also (have tested with web client and things are working) - 
```suggestion
      additionalMediaOptions = {},
```

Reply:
https://app.vidcast.io/share/34f6c35a-aa3f-4932-a60b-7d1e12e87433